doc/rpc.rst

```diff
@@ -44,11 +44,10 @@ The library provides an API to do the following:
 Connecting
 ==========
 
-To connect to dune's RPC server, ``$ dune rpc init`` must be ran. This command
-will initiate a new RPC session communicating over `stdin`, `stdout`. When
-instantiating the ``Client`` functor, a read/write ``Chan.t`` must be provided.
-This ``Chan.t`` value should represent read/write from stdin/stdout
-respectively.
+To connect to dune's RPC server, it needs to be started in watch mode. It is
+possible to use ``dune build --passive-watch-mode`` to start a RPC server which
```

Comment:
```suggestion
possible to use ``dune build --passive-watch-mode`` to start an RPC server which
```
I know it’s strange to have “an” before a consonant, but it takes the sound of the next word. If you’re saying R - P - C, then the “R” is pronounced with a vowel at the beginning. (Not expecting you to remember this, just as an explanation for the suggestion.)